hack/api-docs/requirements.txt

```
@@ -6,7 +6,7 @@ livereload==2.6.3
 Markdown==3.3.6
 MarkupSafe==2.0.1
 mkdocs==1.4.3
-mike==1.1.2
+mike @ git+https://github.com/jimporter/mike@master
```

instead of pointing it to `master`, can we point it to a specific commit?

This feature was introduced in https://github.com/jimporter/mike/commit/06417f58fe4fe7d932dd01a1b8de2d53f1a900f8, current master points to `300593c`, [master](https://github.com/jimporter/mike/commits/master).

```suggestion
mike @ git+https://github.com/jimporter/mike@300593c
```
That way we avoid potentially breaking behaviour introduced by a commit on master.